cmd/root.go

```diff
@@ -6,9 +6,36 @@
 package cmd
 
 import (
+	"context"
+	"log"
+
+	gb "github.com/elastic/elastic-agent-changelog-tool/internal/github"
+	"github.com/google/go-github/v32/github"
+	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
+	"golang.org/x/oauth2"
 )
 
+var GithubClient *gb.GithubClient
+
+func init() {
+	authToken := gb.NewAuthToken(&afero.OsFs{})
```

Comment:
CI is failing because this line forces to load the token from the disk for each invocation of the `cmd` package. This means that during testing will check for a token on disk, and it is correctly not existing.

May you clarify why you added this `init()` function? What was it's purpose?

Reply:
The code no longer uses the `init()` function.